src/math/kronecker.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include <bits/stdc++.h>
+using namespace std;
+
+struct Kronecker {
+    template <class T>
+    static void mul(vector<T>& x, T a, T b, T c, T d) {
+        for (int j = 1; j < (int)x.size(); j <<= 1) {
+            for (int i = 0; i < (int)x.size(); i++) {
```

Comment:
```suggestion
        int n = x.size();
        for (int j = 1; j < n; j <<= 1) {
            for (int i = 0; i < n; i++) {
```

こちらの方がタイプ量が減るように思います！